### PR TITLE
Add Globalize 5.x, Rails 4.2 support.

### PR DESCRIPTION
**Pull request by simi**

### CODE REVIEW

**Comment by simi** [comment]
@parndt do we really need to test against all RDBMSes? We're using one simple `where` in codebase.

**Comment by parndt** [comment]
I doubt it.

**Comment by parndt** [comment]
We definitely shouldn't be running CI with `sqlite3.`

**Comment by simi** [comment]
Actually DB env have no effect.

https://github.com/globalize/globalize-versioning/blob/263e9b7e15254f9e696ab608bc152b67026e1516/test/test_helper.rb#L12

**Comment by parndt** [comment]
Cool, so we can just specify Ruby and Rails version.

**Comment by parndt** [comment]
We should remove 1.9.3, it's past end of life, and we should add 2.1 and 2.2

File: .travis.yml [hunk 1]
```diff
@@ -1,12 +1,11 @@
...
+  matrix:
+    - RAILS=3.2
+    - RAILS=4.0
+    - RAILS=4.1
+    - RAILS=4.2
 rvm:
   - 2.0.0
-  - 1.9.3
+  - 2.1.0
+  - 2.2.0
```
Comment (parndt) [inline .travis.yml]: These are supposed to be specified as `2.1` and `2.2` due to the semantic versioning of Ruby.

2.0.0 should remain as 2.0.0 though as it wasn't included in that.

**Comment by simi** [comment]
@parndt feel free to merge. I'm preparing CHANGELOG and gem release.

**Comment by parndt** [comment]
Cool, can you add the changelog to this PR?

**Comment by simi** [comment]
I'm adding global changelog including 0.1.0.alpha.1 also so I would like to commit it in separate commit.

File: test/test_helper.rb [hunk 2]
```diff
@@ -21,7 +21,6 @@
 DatabaseCleaner.strategy = :truncation
 
 require 'minitest/autorun'
-require 'minitest/colorize'
```
Comment (parndt) [inline test/test_helper.rb]: why?

Comment (simi) [inline test/test_helper.rb]: not compatible with minitest 5

**Pull request merged**